data/json/recipes/recipe_ammo.json

```diff
@@ -27,6 +27,27 @@
   ]
 },{
   "type" : "recipe",
+  "result": "40mm_toxicgas",
+  "category": "CC_AMMO",
+  "subcategory": "CSC_AMMO_BULLETS",
+  "skill_used": "fabrication",
+  "difficulty": 3,
+  "time": 100,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "textbook_gaswarfare", 4, "" ] , [ "manual_launcher", 4 ]],
+  "qualities" : [
+    {"id":"SCREW","level":1,"amount":1}
+  ], "components": [
```

Comment:
Needs some gunpowder for a propellant. 40mm shot and flechette use 40 gunpowder and a single shotgun primer per charge.